devdox/app/repositories/api_key_repository.py

```
@@ -0,0 +1,46 @@
+
+class IApiKeyStore(Protocol):
+    
+    @abstractmethod
+    async def query_for_existing_hashes(
+        self, hash_key_list: List[str]
+    ) -> None | List[str]: ...
+    
+    @abstractmethod
+    async def save_api_key(self, create_model: APIKeyCreate) -> Any: ...
+
+class TortoiseApiKeyStore(IApiKeyStore):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)